map/search_mark.hpp

```diff
@@ -79,18 +77,20 @@ class SearchMarkPoint : public UserMark
 
   bool HasReason() const;
 
-  std::string GetSymbolName() const;
+  std::string const * GetSymbolName() const;
 
-  SearchMarkType m_type{};
-  FeatureID m_featureID;
   // Used to pass exact search result matched string into a place page.
   std::string m_matchedName;
-  bool m_isPreparing = false;
-  bool m_hasSale = false;
-  bool m_isSelected = false;
-  bool m_isVisited = false;
-  bool m_isAvailable = true;
   std::string m_reason;
+
+  FeatureID m_featureID;
+  SearchMarkType m_type{};
```

Comment:
Зачем тут фигурные скобки?

Reply:
default = 0 initialization

Reply:
Нифига не очевидно, это же enum. Напиши лучше явно 
```suggestion
  SearchMarkType m_type = SearchMarkType::Default;
```

Reply:
enum определяется в cpp